aspnetcore/grpc/browser.md

```diff
@@ -20,37 +20,21 @@ It is not possible to call a HTTP/2 gRPC service from a browser-based app. [gRPC
 
 ## Configure gRPC-Web in ASP.NET Core
 
-gRPC services hosted in ASP.NET Core can be configured to support gRPC-Web alongside HTTP/2 gRPC. gRPC-Web does not require any changes to services, the only modification is startup configuration.
+gRPC services hosted in ASP.NET Core can be configured to support gRPC-Web alongside HTTP/2 gRPC. gRPC-Web does not require any changes to services. The only modification is startup configuration.
 
 To enable gRPC-Web with an ASP.NET Core gRPC service:
 
 1. Add a reference to the [Grpc.AspNetCore.Web](https://www.nuget.org/packages/Grpc.AspNetCore.Web) package.
 2. Configure the app to use gRPC-Web by adding `AddGrpcWeb(...)` and `UseGrpcWeb()` to *Startup.cs*:
 
-```csharp
-public void ConfigureServices(IServiceCollection services)
-{
-    services.AddGrpc();
-}
+[!code-csharp[](~/grpc//browser/sample/Startup.cs?name=snippet_1&highlight=4,11,15)]
```

Comment:
Check the build. This is not showing up. First thing you do is check for build warnings.

Reply:
You've got `//` in the path.